diff --git a/src/plugins/_core/settings.tsx b/src/plugins/_core/settings.tsx
index d58c7a98..ca5014d3 100644
--- a/src/plugins/_core/settings.tsx
+++ b/src/plugins/_core/settings.tsx
@@ -24,6 +24,7 @@ import PluginsTab from "@components/VencordSettings/PluginsTab";
 import ThemesTab from "@components/VencordSettings/ThemesTab";
 import UpdaterTab from "@components/VencordSettings/UpdaterTab";
 import VencordTab from "@components/VencordSettings/VencordTab";
+import VendroidTab from "@components/VencordSettings/VendroidTab";
 import { Devs } from "@utils/constants";
 import { getIntlMessage } from "@utils/discord";
 import definePlugin, { OptionType } from "@utils/types";
@@ -124,6 +125,12 @@ export default definePlugin({
                 element: BackupAndRestoreTab,
                 className: "vc-backup-restore"
             },
+            {
+                section: "VendroidSettings",
+                label: "Vendroid Settings",
+                element: VendroidTab,
+                className: "vc-vendroid"
+            },
             IS_DEV && {
                 section: "VencordPatchHelper",
                 label: "Patch Helper",
